### PR TITLE
Remove Chromium notes for navigator.language/languages

### PR DESCRIPTION
**Pull request by foolip**
See https://github.com/mdn/browser-compat-data/pull/8035#discussion_r546213534

### CODE REVIEW

File: api/NavigatorLanguage.json [hunk 1]
```diff
@@ -107,16 +102,13 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/NavigatorLanguage/languages",
           "support": {
             "chrome": {
-              "version_added": "37",
```
Comment (foolip) [inline api/NavigatorLanguage.json:110]: This bug was real and tracked by https://bugs.chromium.org/p/chromium/issues/detail?id=351715, fixed in Chromium 65: https://storage.googleapis.com/chromium-find-releases-static/a02.html#a029f27a6e98a43e8f1492b85ca18007670d9b28

We could keep a note, but does it belong on `navigator.language` or `navigator.languages`?

I don't know what a web developer would do with this information, so my default suggestion is to remove it.

**Comment by foolip** [comment]
@ddbeck @Elchi3 your input on these notes would be great. There are a couple of issues I think:
- Do `navigator.languages` and `Accept-Language` match each other and the browser or OS languages settings?
- Is `navigator.languages === navigator.languages[0]` guaranteed?

The PR is to remove all the notes, but I'm not committed to that, I just wanted to raise it because *some* change is definitely needed to reflect the current reality.

**Comment by ddbeck** [comment]
> * Do `navigator.languages` and `Accept-Language` match each other and the browser or OS languages settings?

My take: no notes for any of this.

When it comes to browser and OS settings relationships' to anything else, I don't think we should have a note. The relevant specs don't call for a specific way to get the user's language preference (e.g., built-in or system UI). The HTML spec even suggests that a browser should override the user's preference, to prevent fingerprinting.

As for the relationship between `navigator.languages` and `Accept-Language`, I also think we shouldn't bother with a note, but I'm less certain. It might be surprising if they're different, but the the HTML spec says that `navigator.languages` and `Accept-Language` should be the same (again) to prevent fingerprinting. That doesn't sound like something that was written in as a guarantee to web developers.

> * Is `navigator.languages === navigator.languages[0]` guaranteed?

That seems like the only noteworthy part of this, though I won't be bothered if we ignore it and move on. The spec speaks directly to web developers on this point ([in the non-normative box here](https://html.spec.whatwg.org/multipage/system-state.html#language-preferences)): "The most preferred language is the one returned by `navigator.language`." I like the idea of letting the spec lead us here: if `navigator.language !== navigator.languages[0]` then we should note that on `navigator.languages` and not on `navigator.language`. Something like "Before Chrome 65, `navigator.languages[0]` was not guaranteed to match `navigator.language`."

I don't feel strongly about this though. Interested to hear what Florian has to say.

**Comment by Elchi3** [comment]
"Before Chrome 65, `navigator.languages[0]` was not guaranteed to match `navigator.language`."
Yes, This sounds like a typical note that would appear in BCD as it describes a change in behavior by a user agent.

The other (non-)guarantees probably belong in the documentation rather?

**Comment by foolip** [comment]
OK, I'll take a stab at keeping and updating the notes about `navigator.languages[0]` and `navigator.language`. Not as a high priority though.

File: api/NavigatorLanguage.json [hunk 2]
```diff
@@ -107,16 +102,13 @@
...
-              "notes": "In Chrome, <code>navigator.language</code> is the language of the browser UI, and is not guaranteed to be equal to <code>navigator.languages[0]</code>."
+              "version_added": "37"
```
Comment (ddbeck) [inline api/NavigatorLanguage.json:105]:
```suggestion
              "version_added": "37",
              "notes": "Before Chrome 65, <code>navigator.languages[0]</code> is not guaranteed to equal <code>navigator.language</code>."
```

File: api/NavigatorLanguage.json [hunk 3]
```diff
@@ -107,16 +102,13 @@
...
             },
             "chrome_android": {
-              "version_added": "37",
-              "notes": "In Chrome, <code>navigator.language</code> is the language of the browser UI, and is not guaranteed to be equal to <code>navigator.languages[0]</code>."
+              "version_added": "37"
```
Comment (ddbeck) [inline api/NavigatorLanguage.json:108]:
```suggestion
              "version_added": "37",
              "notes": "Before Chrome 65, <code>navigator.languages[0]</code> is not guaranteed to equal <code>navigator.language</code>."
```

File: api/NavigatorLanguage.json [hunk 4]
```diff
@@ -143,12 +135,10 @@
...
             },
             "samsunginternet_android": {
-              "version_added": "3.0",
-              "notes": "In Samsung Internet, <code>navigator.language</code> is the language of the browser UI, and is not guaranteed to be equal to <code>navigator.languages[0]</code>."
+              "version_added": "3.0"
             },
             "webview_android": {
-              "version_added": "37",
-              "notes": "In Chrome, <code>navigator.language</code> is the language of the browser UI, and is not guaranteed to be equal to <code>navigator.languages[0]</code>."
+              "version_added": "37"
```
Comment (ddbeck) [inline api/NavigatorLanguage.json:141]:
```suggestion
              "version_added": "37",
              "notes": "Before version 65, <code>navigator.languages[0]</code> is not guaranteed to equal <code>navigator.language</code>."
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
Added notes as actual suggestions here.

**Comment by ddbeck** [comment]
Hey @foolip, if there's anything I can do to help this along, let me know.

**Comment by ddbeck** [comment]
@foolip Let me know if you plan to revisit this PR. Otherwise, I'll move ahead by applying my suggestions and merging it for the release on Thursday.

**Comment by foolip** [comment]
@ddbeck it would be great if you can tweak the notes to your liking, sorting this out was more work than I had hoped :)

**Review by ddbeck — APPROVED** [review]

**Comment by ddbeck** [comment]
@foolip Thank you!

**Pull request merged**